### PR TITLE
Update net.lutris.Lutris.metainfo.xml

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

**Review by strycore — APPROVED** [review]

**Comment by strycore** [comment]
I never noticed there was a link to diolinux in there!

File: share/metainfo/net.lutris.Lutris.metainfo.xml [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
   <project_license>GPL-3.0</project_license>
-  <metadata_license>GPL-3.0</metadata_license>
+  <metadata_license>CC0-1.0</metadata_license>
   <translation type="gettext">lutris</translation>
   <developer_name>Lutris Team</developer_name>
   <update_contact>strider@lutris.net</update_contact>
   <name>Lutris</name>
-  <summary>Lutris is an open source gaming platform</summary>
+  <summary>Play all your games by Lutris</summary>
+  <summary xml:lang="pt_BR">Jogue todos os seus jogos pelo Lutris</summary>
```
Comment (stephanlachnit) [inline share/metainfo/net.lutris.Lutris.metainfo.xml:12]: These translations should be done via Meson, not implemented directly.

File: share/metainfo/net.lutris.Lutris.metainfo.xml [hunk 2]
```diff
@@ -26,7 +26,7 @@
   <url type="bugtracker">https://github.com/lutris/lutris/issues</url>
   <launchable type="desktop-id">net.lutris.Lutris.desktop</launchable>
   <releases>
-    <release date="2020-07-18" version="0.5.7.1" />
+    <release date="2020-10-19" version="0.5.8-beta" />
```
Comment (stephanlachnit) [inline share/metainfo/net.lutris.Lutris.metainfo.xml:29]: Why was this done? There is no official beta afaik.

File: share/metainfo/net.lutris.Lutris.metainfo.xml [hunk 3]
```diff
@@ -3,21 +3,21 @@
 <component type="desktop">
   <id>net.lutris.Lutris</id>
   <project_license>GPL-3.0</project_license>
-  <metadata_license>GPL-3.0</metadata_license>
+  <metadata_license>CC0-1.0</metadata_license>
```
Comment (stephanlachnit) [inline share/metainfo/net.lutris.Lutris.metainfo.xml:6]: If you change this, please change it accordingly in `debian/copyright`.

Comment (strycore) [reply share/metainfo/net.lutris.Lutris.metainfo.xml:6]: I don't know why this was changed. I don't want licenses other than GPL-3 in the project.

**Pull request merged**